index.js

```diff
@@ -0,0 +1,12 @@
+/* jshint node: true */
+'use strict';
+
+module.exports = {
+  name: 'ember-load-initializers',
+
+  included: function() {
+    this._super(...arguments);
```

Comment:
This executes in Node, where we may not have spread arguments.

I'd suggest using `this._super.apply(this, arguments);` instead.